tests/test_utils.py

```diff
@@ -1,24 +1,24 @@
-import pyfar
+# import pyfar
 
 
-def test_copy(sphericalvoronoi, timedata, frequencydata):
```

Comment:
How do we go about this?
- rename this to `test_copy`
- move the test into the class tests, e.g., `test_coordinates::test_copy()`

Reply:
renamed to test copy for now